source/data-formats.txt

```diff
@@ -21,7 +21,7 @@ Specialized Data Formats
    :titlesonly:
    :maxdepth: 1
 
-   /data-formats/time-series
+   Time Series Data </data-formats/time-series>
```

Comment:
Hey @lindseymoore, do you think we need to repeat "Data" between the heading and the label since it's just the 2 in close proximity?

Reply:
Good point, will remove Data!